src/components/LoginCard/LoginCard.js

```diff
@@ -40,37 +41,37 @@ const onFailure = repsonse => {
   console.log('Problem sign in');
 };
 
-const LoginContainer = ({history}) => (
+const LoginContainer = ({ history }) => (
   <ApolloConsumer>
     {client => (
-      <Button.Group vertical size="large">
+      <Button.Group fluid textAlign="center">
         <Button
+          className="button--login"
           onClick={() => auth.login()}
-          positive
-          icon
-          labelPosition="right"
+          size="large"
+          color="black"
         >
-          Login with Auth0
-          <Icon name="chevron right" />
+          <img className="auth0-logo" src={Auth0_logo} alt="Auth0 logo" />
```

Comment:
It looks like we can get the shield logo alone [here](https://styleguide.auth0.com/resources/logos). We could use this sans the auth0 text to have things align more naturally and be more consistent with all the other [buttons](https://styleguide.auth0.com/resources/logos).
This would be a good extension to make to the Semantic `<Button />` component.

Reply:
The `shield` icon that ships with semantic is also decently close and has consistent styling with other icons.

Reply:
decently close is not good enough for me